tests/integrations/celery/test_celery.py

```diff
@@ -245,7 +241,10 @@ def dummy_task():
     assert results == [42] * 10000
 
     with configure_scope() as scope:
-        assert not scope._tags
+        # Because celery is executed with "task_always_eager" the task is not sent to the
```

Comment:
so does this mean the celery behavior has changed compared to before? 
will leave another comment on the celery integration regarding `push_scope`.